### DISCUSSION TITLE
Discounts for group/bulk purchase

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#211] [Ideas]
I wonder if there is any demand for group discounts? I if you purchase more than X number of tickets we will give you X% or £X off each ticket? I'm thinking current work around would be to give a promo code to those I know book in bulk, but that then doesn't help those new people. 

Any helpful thoughts on future solutions and workarounds appreciated.

**Comment by daveearley** [comment]
That's for sure something I could see being added in the future. Tito have similar functionality:
<img width="558" alt="Screenshot 2024-09-19 at 18 40 12" src="https://github.com/user-attachments/assets/a4647241-03e1-4835-9203-3e9751a6b2a8">

**Reply by justynpride** [reply]
That looks like a great way to achieve the discount.
